### ISSUE TITLE
White text on white background in mobile layout 

### ISSUE THREAD

**Issue by Fodoj** [mattermost/mattermost-server#11178]
#### Summary

Since 5.9 or 5.10 something is not right with CSS. If I view our Mattermost via mobile browser, then right sidebar (not the one with list of users) is just white. Text has white colour, background too.

#### Steps to reproduce

Version 5.11.

Try with this theme:

```
{"sidebarBg":"#ffffff","sidebarText":"#593a4e","sidebarUnreadText":"#f59937","sidebarTextHoverBg":"#e2dce0","sidebarTextActiveBorder":"#f59937","sidebarTextActiveColor":"#593a4e","sidebarHeaderBg":"#593a4e","sidebarHeaderTextColor":"#ffffff","onlineIndicator":"#f59937","awayIndicator":"#e2dce0","dndIndicator":"#f74343","mentionBj":"#f59937","mentionColor":"#ffffff","centerChannelBg":"#ffffff","centerChannelColor":"#222222","newMessageSeparator":"#f59937","linkColor":"#f59937","buttonBg":"#f59937","buttonColor":"#ffffff","errorTextColor":"#f74343","mentionHighlightBg":"#f59937","mentionHighlightLink":"#ffffff","codeTheme":"github","mentionBg":"#f59937"}
```

#### Expected behavior

Black text over white background would be nice. Or any text color than white, so that links are visible.

#### Observed behavior (that appears unintentional) 

If I change general sidebar background colour, then it is applied to right sidebar too. But I do want to keep background white, just text in right sidebar needs to have different colour.

**Comment by amyblais** [comment]
Thank you, opened a ticket here: https://mattermost.atlassian.net/browse/MM-16367.

**Comment by amyblais** [comment]
Closing as this has been fixed for upcoming v5.14 release, which will be released on August 16th.
